Dockerfile

```diff
@@ -14,7 +14,7 @@ LABEL org.label-schema.vcs-ref=${VCS_REF}
 LABEL org.label-schema.build-date=${BUILD_DATE}
 
 # Clone autoware repo to access messages
-RUN cd /home/carma/ && git clone https://github.com/usdot-fhwa-stol/autoware.ai.git --depth 1 --branch release/jetta-gillet
+RUN cd /home/carma/ && git clone https://github.com/usdot-fhwa-stol/autoware.ai.git --depth 1 --branch carma-system-4.0.1
```

Comment:
4.0.0? There is no 4.0.1 tag on autoware

Reply:
Good catch, that was before I sent that message, I was thinking I'd have to do a new 4.0.1 hotfix for autoware as well (and anything else that included msgs)